Library/PhotosCollectionLayout.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+class PhotosCollectionLayout: UICollectionViewFlowLayout {
```

Comment:
Since this class uses the #if thingy to distinguish between iOS and other I think it might be useful to create separate flowLayout classes for both the targets, so you only need this #if statement when initialising the CollectionView with the needed layout. Will be easier to read and maintain!